apps/trading/components/welcome-dialog/get-started.tsx

```diff
@@ -94,7 +84,7 @@ export const GetStarted = ({ lead }: Props) => {
     { 'mt-8': !lead }
   );
 
-  if (getStartedNeeded) {
```

Comment:
user theoretically can fulfill all onboarding condition with no "dismiss" click. In such a case we still will show him get started banner

Reply:
Personally I think this is fine. It gets very complicated if we conditionally show based on all the user state (accounts/deposits/connection/etc). I think the better approach here is to only track the dismissed flag. With the current flow after you deposit you are presented with 'Ready to trade' which when clicked will fully dismiss onboarding

![Screenshot 2023-09-14 at 11 15 01](https://github.com/vegaprotocol/frontend-monorepo/assets/6803987/09a9c619-6641-4826-8d41-17a665ecf7df)